README.rst

```diff
@@ -43,15 +43,15 @@ Some noteworthy flags::
 
 The ``-w`` flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.::
 
-    codespell -I FILE, --ignore-words=FILE
+    codespell -I, --ignore-words FILE
```

Comment:
I assume the original style with an equals worked here? I'd probably favour reverting the change of this line so it still matches grep's man page.

Reply:
Redone to adhere to grep(3) syntax.